### DISCUSSION TITLE
Update ReadMe with the following please;

### DISCUSSION THREAD

**Discussion by shoodidagen** [chaifeng/ufw-docker#113] [General]
I assumed the installer would understand the Subnets my Ubuntu 22.04 system is using, but it added the following to after.rules;
192.168.0.0/16
10.0.0.0/8
172.16.0.0/12

Could we please update to double check the correct subnets are being input when editing /etc/ufw/after.rules

The following shows all system subnets in CIDR - this could be very helpful to others
`ip -o -4 route show | awk '{print $1}' | grep -oP '\b\d+\.\d+\.\d+\.\d+\/\d+\b'`
![image](https://github.com/chaifeng/ufw-docker/assets/78660514/c4f7487c-6473-413e-a7d9-2788f4fb5bc3)

Then adjust the ufw-docker example to suit;

![image](https://github.com/chaifeng/ufw-docker/assets/78660514/26f725fd-20f3-4d80-8a31-70416ef07088)

**Comment by vfhky** [comment]
> I assumed the installer would understand the Subnets my Ubuntu 22.04 system is using, but it added the following to after.rules;
> 
> 192.168.0.0/16
> 
> 10.0.0.0/8
> 
> 172.16.0.0/12
> 
> 
> 
> Could we please update to double check the correct subnets are being input when editing /etc/ufw/after.rules
> 
> 
> 
> The following shows all system subnets in CIDR - this could be very helpful to others
> 
> `ip -o -4 route show | awk '{print $1}' | grep -oP '\b\d+\.\d+\.\d+\.\d+\/\d+\b'`
> 
> ![image](https://github.com/chaifeng/ufw-docker/assets/78660514/c4f7487c-6473-413e-a7d9-2788f4fb5bc3)
> 
> 
> 
> Then adjust the ufw-docker example to suit;
> 
> 
> 
> ![image](https://github.com/chaifeng/ufw-docker/assets/78660514/26f725fd-20f3-4d80-8a31-70416ef07088)
> 
> 

感觉是个好思路，还有种情况就是如果后面又有新的容器使用了新的自定义网络，这种又该如何处理？
